packages/vuetify/src/components/VRating/_variables.scss

```diff
@@ -1,5 +1,15 @@
-@import '../../styles/styles.sass';
+$rating-white-space: nowrap !default;
+$rating-item-align-items: center !default;
 
-$rating-padding: 0.5rem !default;
-$rating-border-radius: 50% !default;
-$rating-dense-padding: 0.1rem !default;
+
+$rating-sizes: () !default;
+$rating-sizes: map-deep-merge(
+  (
+    'x-small': 16px,
+    'small': 20px,
+    'default': 24px,
+    'large': 28px,
+    'x-large': 32px,
```

Comment:
VBtn just defines a base size then the others are calculated from `$size-scales` &times; some multiplier

Reply:
will remove, as current implementation just renders v-btns